openapi/components/schemas/CreditMemos/CreditMemoAllocation.yaml

```
@@ -13,23 +13,24 @@ properties:
     allOf:
       - $ref: ../ResourceId.yaml
   amount:
-    description: Amount of credit from the credit memo allocated to the given invoice.
+    description: |-
+      Amount of credit, from the credit memo, that is allocated to a specified invoice.
     type: number
     format: double
   currency:
     readOnly: true
     allOf:
       - $ref: ../CurrencyCode.yaml
   createdTime:
-    description: Credit memo allocation time.
+    description: Date and time at which the credit memo is allocated.
     readOnly: true
     allOf:
       - $ref: ../ServerTimestamp.yaml
   updatedTime:
     $ref: ../UpdatedTime.yaml
   _links:
     type: array
-    description: The links related to the resource.
+    description: Related resource links.
```

```suggestion
    description: Resource-related links.
```

Are these links to resources or links related to the resource? (They're both, in fact, don't know if my version is superior but I find it clearer)